.python-version

```diff
@@ -0,0 +1 @@
+3.10
```

Comment:
We probably don't need to commit this (or should use 3.11)

Reply:
Removed!